.github/workflows/build.yml

```diff
@@ -5,6 +5,7 @@ on:
     - '**'
     - '!doc/contributors/**'
     - '!doc/design/**'
+    - '**/*.md'
```

Comment:
TODO: we should move this "!*" type paths as [path_ignore](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-ignoring-paths) items instead.